optax/_src/utils_test.py

```diff
@@ -61,5 +62,70 @@ def fn(inputs):
         sum(jax.tree_util.tree_leaves(
             jax.tree_util.tree_map(lambda x: x**2, inputs))))
 
+
+class MultiNormalDiagFromLogScaleTest(parameterized.TestCase):
+
+  @staticmethod
```

Comment:
No need to use a @staticmethod here (see https://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators), consider making it a class function.